### ISSUE TITLE
stopping client wont work

### ISSUE THREAD

**Issue by UweK63** [Edzelf/Esp-radio#134]
Hello,
when I use the Stop-command (in Serialmonitor or over webinterface) the server goes into an endless loop with error "D: Stopping client". Only hard-reset is then possible. Thatswhy I can't change preset oder play another station ...
My wiring is like the minimal configuration in pdf. Using NodeMCU and VS1053B.
Any ideas?
Many thanks, Uwe

**Comment by Edzelf** [comment]
I will test it tomorrow.

**Comment by Edzelf** [comment]
You are right.  The latest version of the wifi core libraries has a bug:  wificlient.connected() return true after wificlient.stop().  I 've made a work-around for this.  I will update the sketch.

**Comment by UweK63** [comment]
Super! Many thanks for your fast response.

**Comment by Edzelf** [comment]
WHITE is defined in (version 0.9 of) TFT_ILI9163C.h.
